packages/clerk-js/src/ui/components/UserProfile/AddAuthenticatorApp.tsx

```diff
@@ -43,37 +43,30 @@ export const AddAuthenticatorApp = withCardStateProvider((props: AddAuthenticato
   }
 
   return (
-    <FormContainer headerTitle={title}>
+    <FormContainer
+      headerTitle={title}
+      headerSubtitle={
+        displayFormat == 'qr'
+          ? localizationKeys('userProfile.mfaTOTPPage.authenticatorApp.infoText__ableToScan')
+          : localizationKeys('userProfile.mfaTOTPPage.authenticatorApp.infoText__unableToScan')
+      }
+    >
       {!totp && <FullHeightLoader />}
 
       {totp && (
         <>
           <Col gap={4}>
             {displayFormat == 'qr' && (
-              <>
-                <Text
-                  localizationKey={localizationKeys('userProfile.mfaTOTPPage.authenticatorApp.infoText__ableToScan')}
-                />
-
-                <QRCode url={totp.uri || ''} />
-
-                <Button
-                  variant='link'
-                  onClick={() => setDisplayFormat('uri')}
-                  localizationKey={localizationKeys(
-                    'userProfile.mfaTOTPPage.authenticatorApp.buttonUnableToScan__nonPrimary',
-                  )}
-                />
-              </>
+              <QRCode
+                justify='center'
+                url={totp.uri || ''}
+              />
             )}
 
             {displayFormat == 'uri' && (
               <>
                 <Text
-                  localizationKey={localizationKeys('userProfile.mfaTOTPPage.authenticatorApp.infoText__unableToScan')}
-                />
-
-                <Text
+                  colorScheme='neutral'
```

Comment:
Why is `inherit` not working for us here?

Reply:
This is a different color in the designs, than the rest of the text on this screen.